sysdig/internal/client/v2/list.go

```diff
@@ -93,6 +97,8 @@ func (client *Client) DeleteList(ctx context.Context, id int) error {
 		return client.ErrorFromResponse(response)
 	}
 
+	client.policiesChanged = true
```

Comment:
Why not calling here endpoint that policy is changed? Also same question for other places?

Reply:
Modified to move logic for whether to call the endpoint into the related resources